WordPress/gutenberg
# Prevent jumping of search button

Fixes: #52854


## What?

When the search box has `change button position` set to `button only` and is aligned to the right while having blocks to it left, the search button jumps, highlighted in #52854.
This PR fixes this jumping of search button when the search box expands.

## How?
Removed `flex-basis` property from the input box which was making the button jump.

## Testing Instructions

1. Add/edit existing post.
2. Add a row.
3. Add some blocks on the left side of the row.
4. Add the search block to the right.
5. Set `change button position` to `button only`.
6. On the frontend the search button should no longer jump while expanding the search box.

## Screenshots or screencast 

### Before
https://github.com/user-attachments/assets/22dba0f1-66b1-4693-aadb-30d2d10f086c

### After
https://github.com/user-attachments/assets/35d01373-8c07-4949-8bc9-29023524d504
